AwaMelvine/fast-food-fast
# Fix deployment issue with Heroku 164944913

#### What does this PR do?
Fixes issue with frontend deployment on heroku.

#### Description of Task to be completed?
- Delete node_modules/ directory
- Delete package-lock.json
- Run the `npm install` command
- remove the `--watch` flag in build script

#### How should this be manually tested?
Visit the root directory of the website on Heroku (at http://fast-food-fast-service.herokuapp.com) and check if the webpages are well hosted.
#### What are the relevant pivotal tracker stories?
[#164944913] Fix issue with deployment on Heroku
